.github/workflows/test.yml

```diff
@@ -19,9 +18,18 @@ jobs:
           python-version: '3.8'
       - name: Install dependencies
         run: |
-          pip install $(grep -iE "pylint|pycodestyle" analyzer/requirements_py/dev/requirements.txt)
-      - name: Run tests
-        run: make pylint pycodestyle
+          pip install $(grep -iE \
+            "mypy|pycodestyle|pylint|types" \
+            analyzer/requirements_py/dev/requirements.txt) \
+            $(grep -iE \
```

Comment:
This patch updated `codechecker_common`'s dev-requirements with new dependencies, and those new dependencies are needed to be installed for the static analysis to be run in this linting job. The "common tests" now runs only the unit tests for the new code added to `codechecker_common`. Having static analysis (`mypy`) run together with the other linters is more appropriate than making the hard test case checking fail because `mypy` overapproximated something.